config/routes.rb

```diff
@@ -9,38 +9,45 @@
 
   scope module: :organizations do
     resources :home, only: [:index]
-
-    resource :organization_profile, only: %i[edit update]
-    resource :page_text, only: [:edit, :update]
-
-    resource :adopter_foster_profile, except: :destroy, as: "profile"
-    resources :profile_reviews, only: [:show]
     resources :adoptable_pets, only: [:index, :show]
+    resources :faq, only: [:index]
 
-    resources :pets do
-      resources :tasks
-      post "attach_images", on: :member, to: "pets#attach_images"
-      post "attach_files", on: :member, to: "pets#attach_files"
-    end
-    resources :default_pet_tasks
-    resources :faqs
-    resources :public_faq, only: [:index]
-    resources :dashboard, only: [:index]
-    resources :adopter_foster_dashboard, only: [:index]
-    resources :adopter_applications, path: "applications",
-      only: %i[index create update]
     resources :forms do
```

Comment:
Ah I think I missed this one in my ticket example. This should be in the staff namespace - it's for CRUDing the org forms and questions.